Ansible/ui.yml

```diff
@@ -28,6 +28,8 @@
         - needs_certificate
         - emi_service
         - yaim_options
+        - site_name
+        - ronald
```

Comment:
I don't think you have a ronald `role` here ? You just set a variable, right ? You should remove the `site_name` as well - that's not a role, but a variable. If you want to debug it, set a `pre_task` do do something like : 

```
pre_tasks:
  - name: debug my vars 
    debug: 
      var: "{{ item }}"
   with_items:
        - "{{ ronald }}"
        - "{{ site_name }}"